app/assets/stylesheets/templates/_question.scss

```diff
@@ -6,7 +6,7 @@
     display: block;
     a {
       &:before {
-        content:'←';
+        content:'←' / '';
```

Comment:
[dust] - this syntax is new to me, and is cool! looked it up and here is a link in case anyone is interested:

https://www.w3.org/TR/css-content-3/#alt